src/jimple-frontend/AST/jimple_ast.h

```diff
@@ -1,11 +1,3 @@
-/*******************************************************************\
-Module: Jimple AST Interface
-Author: Rafael Sá Menezes
-Date: September 2021
-Description: This interface will define every method that needs to
```

Comment:
Shall we keep this description?

Reply:
I don't think so, it's not like they add a lot.